packages/e2e-test-app/windows/ReactUWPTestApp.sln

```diff
@@ -46,52 +46,36 @@ Global
 		..\..\..\vnext\Shared\Shared.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|ARM = Debug|ARM
-		Debug|ARM64 = Debug|ARM64
```

Comment:
note: you are removing Arm64 as well... Maybe regex bug?

Reply:
note: multiple removals of arm64 in this file...

Reply:
Good catch